src/commands/env/logdrain/remove.ts

```diff
@@ -75,12 +76,36 @@ export default class LogDrainRemove extends Command {
       this.warn(messages.getMessage('flags.url.deprecation'));
     }
 
-    const appName = await resolveAppNameForEnvironment(targetCompute);
+    try {
```

Comment:
Remove this try/catch as the error should be caught by the plugin-wide error handler (which will eventually be updated to include json)